b827ebfffe773037

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "b827ebfffe773037",
+    "servers": [
+      {
+        "server_address": "http://18.188.120.154",
```

Comment:
The address should not include the protocol, only the IP address or domain name.